sample-clients/csharp/client/ScoringClient.cs

```diff
@@ -41,19 +42,20 @@ public ScoringClient(string host, int port, bool useSsl = false, string authKey
             _client = new PredictionServiceClientWrapper(new PredictionService.PredictionServiceClient(channel));
         }
 
-        public async Task<float[]> ScoreAsync(IScoringRequest request)
+        public async Task<float[]> ScoreAsync(IScoringRequest request, int retryCount = 20)
```

Comment:
Why is it so high?

Reply:
Set in #14 -  I have no idea why 20 was picked. It being so high is why this needs to exist - some of our tests take a long time to time out.